rules.go

```diff
@@ -7,45 +7,82 @@ import (
 
 type NewRuleNoJson NewRule
 
+// AuthenticationMode is an enum of authentication modes used by pulsar rules.
 type AuthenticationMode string
```

Comment:
Should this be `PulsarAuthenticationMode` instead, so that it is clear that it is for Pulsar rules?

Reply:
Yep done